mod_reforged/hooks/msu.nut

```diff
@@ -81,14 +81,23 @@
 }
 
 ::logInfo("Reforged::MSU -- adding ::MSU.Text.colorizeMult");
-::MSU.Text.colorizeMult <- function( _value, _options = null )
+::MSU.Text.colorizeMult <- function( _value, _options = null )		// will turn values like 1.45 into a formatted and colored 45%
 {
 	if (_options == null) _options = {};
 	if (!("AddSign" in _options)) _options.AddSign <- false;
 	_options.AddPercent <- true;
 	return this.colorizeValue((_value - 1.0) * 100, _options);
 }
 
+::logInfo("Reforged::MSU -- adding ::MSU.Text.colorizeFraction");
+::MSU.Text.colorizeFraction <- function( _value, _options = null )	// will turn values like 0.2 into a formatted and colored 20%
```

Comment:
`_value` here must be between 0 and 1, so for completeness sake you could add a requirement for that in the beginning of the function i.e.
```squirrel
if (_value < 0 || _value > 1)
   throw ::MSU.Exception.InvalidValue(_value);
```

Reply:
The Fraction concept I have in mind is allowed to go past 1.0.
A Fraction of 1.3 is equal to 130%. E.g. Fearsome would now can 130% of Resolve as a Penalty which is absolutely allowed

I put for now a restriction for not going lower than 0.0 like you suggested